src/Control/Lens/Internal/ByteString.hs

```diff
@@ -234,41 +215,14 @@ c2w = fromIntegral . ord
 
 -- | Unpack a strict 'B.Bytestring'
 unpackStrict :: B.ByteString -> [Word8]
-#if MIN_VERSION_bytestring(0,10,4)
 unpackStrict = B.unpack
-#else
-unpackStrict (BI.PS fp off len) =
-      let p = unsafeForeignPtrToPtr fp
-       in go (p `plusPtr` off) (p `plusPtr` (off+len))
-    where
-      go !p !q | p == q    = []
-               | otherwise = let !x = BI.inlinePerformIO $ do
-                                        x' <- peek p
-                                        touchForeignPtr fp
-                                        return x'
-                             in x : go (p `plusPtr` 1) q
-#endif
 {-# INLINE unpackStrict #-}
 
 -- | Unpack a strict 'B.Bytestring', pretending the bytes are chars.
 unpackStrict8 :: B.ByteString -> String
```

Comment:
Similarly, is there a good reason not to remove this function and replace its use sites with `B8.unpack`?

Reply:
Ditto.